src/crimson/osd/osd.cc

```diff
@@ -5,6 +5,7 @@
 #include <boost/smart_ptr/make_local_shared.hpp>
 #include <fmt/format.h>
 #include <fmt/ostream.h>
+#include <seastar/core/sleep.hh>
```

Comment:
is `seastar::sleep()` or its friends used here?

Reply:
Nope, I think I added it for debugging, will remove.